src/sqlparser/src/ast/mod.rs

```diff
@@ -743,6 +743,64 @@ impl fmt::Display for CommentObject {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum ExplainType {
+    Logical,
+    Physical,
+    DistSQL,
+}
+
+impl fmt::Display for ExplainType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            ExplainType::Logical => f.write_str("Logical"),
+            ExplainType::Physical => f.write_str("Physical"),
+            ExplainType::DistSQL => f.write_str("DistSQL"),
+        }
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct ExplainOptions {
+    // Display additional information regarding the plan.
```

Comment:
```suggestion
    /// Display additional information regarding the plan.
```

Reply:
In fact, I mean we should make this doc comments for all fields, so all fields should starts with `///`